src/content/docs/inventory/inventory-api.md

```diff
@@ -33,12 +33,23 @@ The request body is the resource type. This top level json object is comprised o
 
 Required data is marked with an asterisk (*).
 
+#### Tenancy
+
+All resources registered in Inventory must contain tenancy information. This information is used to determine in which Kessel workspace the resource will be registered.
+
+Clients of Inventory have two alternatives to provide this tenancy information. If the client sets either _workspace_ or _workspace_id_ in the _metadata_ section, the Inventory will attempt to store the resource in that specific workspace.
```

Comment:
@akoserwal 

> it should be just workspace_id? instead of workspace or workspace_id?

It should be a [Oneof](https://protobuf.dev/programming-guides/proto3/).

> Is there a guarantee that the workspace name can be unique within the same organization?

This is a great question. We have to clarify what are the uniqueness guarantees of workspace names.

Reply:
I seem to remember yesterday @alechenninger suggested we do a `Oneof` of *only* `workspace_id`. And leave `workspace` (by name) for the future.

Reply:
Right now, we are only supporting workspace_id. If there is a requirement to support workspace, then this is how you would implement it. With that said, @akoserwal's question is valid.  What is the namespace in which names are unique?  A multi-tenant agent would need to pass the org_id and the workspace (name) if name is only unique within an org.  The org_id from that agent's token is not adequate.

Reply:
On relationships, these isn't tenancy per-se.  A relationship is created between a subject and object. Each already exists with their respective tenancy.  Currently, we only support a relationship between 2 resources of the same tenant.  The relationship is akin to an attribute. It "inherits" the same tenant as the the resource.

Reply:
OK, in this case, I'm going to simplify this for now, and remove `workspace` from the PR.

Reply:
@randymgeorge regarding relationship tenancy that makes sense. Since the `org_id` was documented as a requirement I assumed it was going to be used in the same way, but what you say makes a lot of sense.